repo.path

```diff
@@ -5,5 +5,9 @@
 # Do not build examples
 !kb/pattern
 
+#processing program
+#!kb/section_subject_domain_of_dialogues/section_subject_domain_of_dialog_control/examples/message_processing_program.gwf
+!kb/section_subject_domain_of_dialogues/section_subject_domain_of_dialog_control/examples/weather_message_processing_program.gwf
```

Comment:
Can we use only one program and call weather agent only if needed? Like InterfaceAgent